http-server/build.gradle

```diff
@@ -1,22 +1,42 @@
 plugins {
     id 'application'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
-    id 'de.undercouch.download' version '3.4.3'
 }
 
 archivesBaseName = "$group-$name"
 description = 'TripleA DropWizard Http Server'
-mainClassName = '' // org.triplea.game.client.HeadedGameRunner'
-
+mainClassName = 'org.triplea.server.http.ServerApplication'
 ext {
     releasesDir = file("$buildDir/releases")
 }
 
+
+configurations {
+    testImplementation {
+        exclude group: 'org.slf4j', module: 'slf4j-simple'
+    }
+}
+
 dependencies {
-    implementation 'io.dropwizard:dropwizard-core:1.3.11'
     implementation project(':java-extras')
     implementation project(':http-clients')
+
+    implementation 'io.dropwizard:dropwizard-core:1.3.12'
+    implementation 'io.dropwizard:dropwizard-jdbi3:1.3.12'
+    implementation 'org.jdbi:jdbi3-core:3.8.2'
+    implementation 'org.jdbi:jdbi3-sqlobject:3.8.2'
+
+    implementation 'javax.xml.bind:jaxb-api:2.3.1'
```

Comment:
there was a nasty startup warning displayed when drop wizard started. It comes down to jaxb being packaged with JDK8 and before, but no longer with JDK8+. The warnings did not necessarily stop the server from starting, regardless having these dependencies fixes the warnings and is one less thing to worry about.